### PR TITLE
[Recure]: eventType added

### PR DESCRIPTION
**Pull request by Abnormaltype**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,8 +7,19 @@ First, run `npm install recure` for your app.
...
 ```typescript
-import { recure } from 'recure/web'
+import { recure, EventType } from 'recure/web'
 
-// Put this code after successful submitted form sign_up/login
-await recure("userId", "your-project-api-key", "eventType");
-```
\ No newline at end of file
+// Put this code after successful submitted form sign_up/login, 4-th parameter is optional and uses only with EventType.PAGE
+await recure("userId", "your-project-api-key", EventType.NameOfEvent, "subscription-page");
```
Comment (danylott) [inline README.md:13]: add 2 examples - 1 for LOGIN, 1 for PAGE. Do not use `NameOfEvent` here

File: README.md [hunk 2]
```diff
@@ -7,8 +7,19 @@ First, run `npm install recure` for your app.
...
 
 ```typescript
-import { recure } from 'recure/web'
+import { recure, EventType } from 'recure/web'
 
-// Put this code after successful submitted form sign_up/login
-await recure("userId", "your-project-api-key", "eventType");
-```
\ No newline at end of file
+// Put this code after successful submitted form sign_up/login, 4-th parameter is optional and uses only with EventType.PAGE
```
Comment (danylott) [inline README.md:12]: Remove this comment

File: README.md (hunk 1 above)
Comment (danylott) [reply README.md:13]: For Login add comment, that `Put this code after successful submitted form sign_up/login`

**Review by danylott — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Abnormaltype) [reply README.md]: fixed, thanks

**Pull request merged**